typescript/create-onchain-agent/templates/actionProvider/schema.njk

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+/**
+ * Input schema for {{ className }}'s {{ actionName }} action.
+ */
+export const {{ className }}Schema = z
+  .object({
+    payload: z.string().describe("The payload to send to the action provider"),
+  })
+  .strip()
+  .describe("Instructions for {{ actionName }}"); 
```

Comment:
there's an extra whitespace at the end of this line. let's remove it, and also add a blank line to the end of the file